README.md

```diff
@@ -164,6 +164,10 @@ Is semi pretty too + custom. All unittested ...
   "networkd.wg0.ipv6_address_state": 3,
   "networkd.wg0.oper_state": 9,
   "networkd.wg0.required_for_online": 1,
+  "pid1.cpu_time_kernel": 48,
+  "pid1.cpu_user_kernel": 41,
+  "pid1.memory_usage_bytes": 19165184,
+  "pid1.num_threads": 1,
```

Comment:
From v254 onwards pid1 will never use more than a single thread so not sure this one is very useful. Maybe processes in the same cgroup could be useful to track?